src/juju/watchers.ts

```diff
@@ -39,19 +45,79 @@ export function processDeltas(
     return;
   }
   deltas.forEach((delta) => {
-    // The delta messages are in the format [entity, action, data].
+    // Delta is in the format [entityType, actionType, data].
```

Comment:
```suggestion
    // Delta is in the format of [entityType, actionType, data].
```

Reply:
Or you could map it to these object names to make the code more readable?

Reply:
I applied your change. I had also attempted using destructuring but there were issues with TypeScript. See the discussion [here](https://github.com/canonical-web-and-design/jaas-dashboard/pull/1110#discussion_r688669051) for more information.